src/DigiTicketsVoucherGateway.php

```diff
@@ -12,6 +12,12 @@
 
 class DigiTicketsVoucherGateway extends AbstractVoucherGateway
 {
+    /** @var callable*/
```

Comment:
@jamesgarton Technically it should be

```
/** @var callable|null */
```

and there is no need to assign null, as I believe null is assigned by default.